Kernel/Syscalls/pipe.cpp

```diff
@@ -43,7 +43,7 @@ ErrorOr<FlatPtr> Process::sys$pipe(Userspace<int*> pipefd, int flags)
             reader_fd_allocation.fd,
             writer_fd_allocation.fd,
         };
-        if (copy_to_user(pipefd, fds_for_userspace, sizeof(fds_for_userspace)).is_error()) {
```

Comment:
(iʼd even extract that 2 and use it in L42… 😅 – btw. very similar stuff happening in the other changed place for sockets… 🤔)

Reply:
Are you asking me to create a variable like `nr_fds = 2` instead of using the hardcoded 2 directly?
I don't completely understand the `extract 2` comment.

Reply:
yeah, something along those lines (a `constexpr` or something); though this is a minor, not really worth time—up to you—might help to understand where it came from from the first glance.

Reply:
@konradekk Please stop spamming all the PRs with low-value/nitpicking review comments